networkit/_NetworKit.pyx

```diff
@@ -4715,7 +4726,15 @@ cdef class GraphTools:
 	@staticmethod
 	def getContinuousNodeIds(Graph graph):
 		"""
-			Computes a map of node ids to continuous node ids.
+                Computes a map of node ids to continuous node ids.
+
+                Parameters
```

Comment:
Also here, and in other parts of the file. In vim you can use the 'set list' command to see where you are using spaces/tabs.